README.md

```diff
@@ -3,4 +3,88 @@
 [![Build Status](https://travis-ci.org/Cimpress-MCP/auth0-sso-login.js.svg?branch=master)](https://travis-ci.org/Cimpress-MCP/auth0-sso-login.js) 
 [![npm version](https://badge.fury.io/js/auth0-sso-login.svg)](https://www.npmjs.com/package/auth0-sso-login)
 
-A javascript package to simplify the auth0 sso login process
+The Auth0 SSO Login provides an easy to use library for single-sign on web pages that are leveraging [Auth0](https://auth0.com/) and related [Auth0 Lock](https://auth0.com/lock).
+
+## Using this library
+
+```javascript
+// import library
+import Auth from 'auth0-sso-login.js';
+
+// create an instance of Auth
+let config = { /* ... */ };
+let auth = new Auth(config);
+
+// optionally call "stay logged in" which is a background process
+// that ensures hourly that the user stay logged in
+auth.stayLoggedIn();
+
+// verifies that the user is logged in
+// returns a promise, when succeeded, the user is logged in
+// and a valid JWT was provided.
+auth.ensureLoggedIn()
+.then(() => console.log('user is logged in'));
```

Comment:
I think we throw some errors too, it would be good to document what a rejection looks like.